sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -977,6 +982,12 @@ class Analyzer(override val catalogManager: CatalogManager)
           .map(ResolvedTable(catalog.asTableCatalog, ident, _))
           .getOrElse(u)
 
+      case u @ UnresolvedView(NonSessionCatalogAndIdentifier(catalog, ident), cmd) =>
+        u.failAnalysis(
```

Comment:
this seems like something that we should put in `CheckAnalysis`

Reply:
Moved to `CheckAnalysis`.